engine/e_bluefield.c

```diff
@@ -425,6 +486,39 @@ static int bind_pka(ENGINE *e)
 #endif
 # endif
 
+#if (OPENSSL_VERSION_NUMBER >= 0x10100000L)
+#ifndef NO_EC
+    if (!engine_pka_pkey_meth_nids_init())
+    {
+        printf("ERROR: PKA ENGINE: nids_init() failed\n");
```

Comment:
I noticed that other error print statements are formatted differently. Any reason we add "PKA ENGINE" in the error print statement?

Reply:
Nice catch! 
I have updated the format of all the print statements to: "ERROR: Function_Name: Error Message". Does this makes sense ?

Reply:
Absolutely

Reply:
Fyi, you can use __func__ in the print statement, for example:

printf("ERROR: %s: msg goes here\n", __func__);

Reply:
Ah, yes, this is better, I will update it.